src/libraries/System.Reflection.Metadata/tests/Metadata/MetadataReaderTests.cs

```diff
@@ -2660,6 +2660,21 @@ public void DebugMetadataHeader()
             }
         }
 
+        [Fact]
+        public void GetMethodDebugInformation_InvalidHandle()
+        {
+            using (var provider = MetadataReaderProvider.FromPortablePdbStream(new MemoryStream(PortablePdbs.DocumentsPdb)))
+            {
+                var reader = provider.GetMetadataReader();
+                var mdi = reader.GetMethodDebugInformation(default(MethodDebugInformationHandle));
+
+                AssertEx.Throws<BadImageFormatException>(() => mdi.SequencePointsBlob, _ => { });
```

Comment:
I do not see the point in using the Ex version here. Can this be a regular `Assert.Throws` without the second arg?

Reply:
Agreed. In my sleepy stupor I thought this file was only using AssertEx and wanted to be consistent... obviously that's not the case.